substrate/frame/asset-conversion/src/lib.rs

```diff
@@ -725,14 +725,14 @@ pub mod pallet {
 	}
 
 	impl<T: Config> Pallet<T> {
-		/// Swap exactly `amount_in` of asset `path[0]` for asset `path[1]`.
+		/// Swap exactly `amount_in` of asset `path[0]` for asset `path[last]`.
```

Comment:
I think we should precise that it swaps 0 with 1 then 1 with 2 until last no?

but I don't know how to phrase it.

Reply:
I think specifying somewhere that assets are swapped through every hop in the path would help users. You can do it if you want